src/app/screens/MakeInvoice.tsx

```diff
@@ -32,32 +32,16 @@ function MakeInvoice({ invoiceAttributes, origin }: Props) {
   );
   const [error, setError] = useState("");
 
-  function handleValueChange(e: React.ChangeEvent<HTMLInputElement>) {
+  function handleValueChange(amount: string) {
```

Comment:
in this one we call `setValue` now with a string instrad of a number. is this fine?

Reply:
e.target.value is always a string, no matter what you pass into an input.

Reply:
Yes, utils.call("makeInvoice"), which will use value as an argument, can handle both strings / numbers.